src/app/views/about/about.view.html

```diff
@@ -0,0 +1,19 @@
+<ng-vertical-timeline [data]="events"></ng-vertical-timeline>
+<h1>My Hobbies & Interests</h1>
+<mat-grid-list cols="4" rowHeight="100px">
+  <mat-grid-tile
+      *ngFor="let tile of tiles"
+      [colspan]="tile.cols"
+      [rowspan]="tile.rows"
+      [style.background]="tile.color">
+  <div *ngIf="tile.text; else elseBlock">{{tile.text}}</div>
+  <ng-template #elseBlock><img src={{tile.image}} class= image></ng-template>
```

Comment:
Class names need to be in quotes, preferably double quotes.
```suggestion
  <ng-template #elseBlock><img [src]="tile.image" class="image"></ng-template>
```